includes/controller/user_settings_controller.php

```diff
@@ -193,6 +194,41 @@ function user_settings() {
         redirect(page_link_to('user_settings'));
       }
   }
+  elseif (isset($_REQUEST['update'])) {
+    $ok = true;
+
+    $online_ver = file_get_contents("https://raw.githubusercontent.com/DishantK1807/engelsystem/auto-update/Version.txt");
+    $current_ver = file_get_contents(" ./Version.txt");
+    if (strcmp($current_ver, $online_ver) != 0) {
+      echo shell_exec("sudo sh ./update.sh");
```

Comment:
it took in the terminal, the second time the script was run. Although it is not required. I'll remove it.